jenkins-runtime/Jenkinsfile

```diff
@@ -7,7 +7,8 @@ properties([
         string(name: 'OSSIM_OMS_BRANCH', defaultValue: 'dev', description: 'The branch to use for ossim oms'),
         string(name: 'OSSIM_VIDEO_BRANCH', defaultValue: 'dev', description: 'The branch to use for ossim video'),
         string(name: 'OSSIM_DEEPCORE_BRANCH', defaultValue: 'dev', description: 'The branch to use for ossim deepcore'),
-        string(name: 'OSSIM_BUILDER_VERSION', defaultValue: '1.0.1', description: 'The version of the ossim builder to be used'),
+        string(name: 'OSSIM_IMAGE_LINKER_BRANCH', defaultValue: 'imageLinker', description: 'The branch to use for ImageLinker'),
```

Comment:
does OSSIM_IMAGE_LINKER_BRANCH need to be changed to a different branch for final build?